parsers/BUILD

```diff
@@ -25,10 +39,17 @@ filegroup(
 ### tests
 
 cc_test(
-    name = "x3d_parser_test",
-    srcs = glob(["test/*.cpp"]),
+    name = "rcbe_x3d_parsers_tests",
+    srcs = [
+        "test/parsers_test.cpp",
+        "test/x3d_parser_test.cpp",
+    ],
+    args = [],  # left to try pass the file through this, this will most likely require custom rules
     copts = ["-Iexternal/gtest/include"],
-    data = glob(["test/resources/*"]),
+    data = [
```

Comment:
Consider making this via `http_file` maybe in separate request

Reply:
I think it will be better if some tests of these are run in pipeline, and as such I need this files locally.

Reply:
'Kay